target/linux/imx/cortexa7/config-default

```diff
@@ -11,6 +11,8 @@ CONFIG_CMA_AREAS=7
 # CONFIG_CMA_SYSFS is not set
 CONFIG_CONTIG_ALLOC=y
 # CONFIG_DMA_CMA is not set
+CONFIG_EXTRA_FIRMWARE="imx/sdma/sdma-imx7d.bin"
```

Comment:
@Leo-PL I think we need to switch to sysfs based solution for this subtarget as at least the i.MX6UL/L will require `sdma-imx6q`. I will handle that when adding other devices, let's keep it as it for now.

Reply:
Not only that, this opens a potential can of worms of linking closed-source SDMA firmware in GPL-licensed kernel.

Reply:
To be honest I never looked at license side of these firmware files.

Reply:
Recently at my day job, my team took a look at this, and this was the conclusion - we switched to FIT images to easily bundle it in initramfs instead.

Reply:
@Leo-PL what do you mean by `bundle` here? Include it as a `loadable` binary blob within FIT?